File: angular/src/app/hotel/models/hotel-location-dto.ts


import { EntityDto } from '@abp/ng.core';

export class HotelLocationDto extends EntityDto<string> {
  latitude: number;
  longitude: number;
  hotelId: string;
  hotel: any;
  id: string;

  constructor(initialValues: Partial<HotelLocationDto> = {}) {
    super(initialValues);
  }
}

File: aspnet-core/src/HotelApp.Application.Contracts/Hotels/HotelLocationDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace HotelApp.Hotels
{
    public class HotelLocationDto : EntityDto<Guid>
    {
        public double Latitude { get; set; }
        public double Longitude { get; set; }

        public Guid HotelId { get; set; }
        public HotelDto Hotel { get; set; }
    }



}
